otheatre/src/main/java/com/orange/otheatre/otheatre/controller/ManageEventsController.java

```diff
@@ -42,6 +48,11 @@
     @Autowired
     EventService saveEvent;
 
+    @Autowired
+    CheckPlayEligibility eligibility;
+
+    List<String> myExceptions = new ArrayList<>();
```

Comment:
Check project as an inspiration StudentManagement to take a look for custom exceptions